core/cas-server-core-web/src/main/java/org/apereo/cas/config/CasCoreWebConfiguration.java

```diff
@@ -30,6 +33,24 @@
     @Autowired
     private CasConfigurationProperties casProperties;
 
+    /**
+     * Get messages from one or more property files containing values
+     * that should be exposed to Thyme templates but don't need i18n.
+     * This bean is Autowired into {@link messageSource()}.
+     * @return PropertiesFactoryBean containing all common (non-i18n) messages
+     */
+    @Bean
+    public PropertiesFactoryBean casCommonMessages() {
```

Comment:
I don't see how I could cleanly call bean.setCommonMessages(Properties props) and use a PropertiesFactoryBean without using auto-wiring. (I am using PropertiesFactoryBean to make it easier to add additional properties files in the future). I can see how to do it uncleanly but it would involve calling getObject() on the PropertieFactoryBean (and dealing with checked exception) and I would also have to call afterPropertiesSet() when I was creating the PropertieFactoryBean. Both of those methods are called by Spring if I make it a bean. 

I set the autowired property to required=false so if the CasReloadableMessageBundle was used somewhere that the casCommonMessages bean wasn't available then it wouldn't be fatal.